database/migrate.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+function missing(dep) {
+	try {
+		require.resolve(dep);
+		return false;
+	} catch (err) {
+		if (err.code !== 'MODULE_NOT_FOUND') throw err;
+		return true;
+	}
+}
+
+function shell(cmd) {
+	child_process.execSync(cmd, {stdio: 'inherit', cwd: __dirname});
+}
+
+if (missing('sqlite')) {
+	shell('npm install sqlite');
+}
+
+const child_process = require('child_process');
+const fs = require('fs');
+const sqlite = require('sqlite');
+
+if (missing('./database/sqlite.db')) {
```

Comment:
Oh, you're actually using `missing` here... Its not a module though? Does this work? Is this to avoid using `fs.existsSync` or whatever? I'm not sure about this trick

Reply:
(Using `missing` didn't actually work here.)